ui/imports/shared/popups/addaccount/panels/AccountAddressSelection.qml

```diff
@@ -124,26 +135,26 @@ StatusMenu {
                     rowSpacing: 0
 
                     StatusBaseText {
-                        Layout.preferredWidth: 208
+                        Layout.fillWidth: true
+                        elide: Text.ElideMiddle
```

Comment:
@caybro I am not sure if is it important to have the same elide across all addresses, like always displaying for each address the first X and the last Y characters, cause with this, that's volatile, for example these two will be displayed like `0x8888888...888888`  and `0x111111111...111111111`.

Reply:
Anyhow, that's so irrelevant, and I like the change you made. :)